### ISSUE TITLE
Custom keybase prove service

### ISSUE THREAD

**Issue by qntmpkts** [keybase/client#2860]
Is there unwritten documentation on custom services to prove or will this be expanded at some point? While it would be trivial to sign a message proving x,y,z, I was wondering if there may be any means of doing this within the keybase CLI. Also is there a technical limitation in place for the number of proved services per account?

**Comment by cjb** [comment]
Which service? It needs to be one where members post public statements that all Keybase users can download.

**Comment by qntmpkts** [comment]
Right, so following those obvious requirements (so keybase can periodically vouch), is there to way to add any arbitrary service, like Facebook or any other public forum a keybase user might frequent?

**Comment by qntmpkts** [comment]
I noticed that if I tried to prove facebook.com/account, the prove command doesn't recognize it.

Perhaps the prove service should be updated to allow any URL and check against the domain.tld/whatever the user specifies and then allot a limited number of services keybase would allow users to prove. Not sure if that is in the pipeline, but sounds like a pretty good idea. Grepping from any URL should be fairly trivial given general enough rules for pattern matching (if that's really important).
